src/components/interface/Button/Button.md

```diff
@@ -62,14 +62,21 @@ import ButtonGroup from './ButtonGroup';
     size="sm"
     isEquisized
     align="center"
-    isInlineFrom="md">
-  <Button
-    secondary 
-    title="title">secondary sm button
-  </Button>
-  <Button
-    title="title">sm button
-  </Button>
+    isInlineFrom="md"
```

Comment:
InlineFrom? responsiveInline?

Reply:
i didn't change the name
and i think it make sens according to the dsfr name: 
![image](https://user-images.githubusercontent.com/5389328/130820210-3811380a-f443-4ef6-8221-f7a4df78debd.png)